branch-from-tag.sh

```diff
@@ -45,7 +45,7 @@ branch_version="${major_version}.${minor_version}.${branch_bugfix_version}-SNAPS
 sed -i "s/${tag_version}/${branch_version}/g" gradle.properties
 
 # Update the Jenkins job default branch name
-sed -i "s/defaultValue: 'master'/defaultValue: '${branch_name}'/g" Jenkinsfile.release
+sed -i "s#defaultValue: 'master'#defaultValue: '${branch_name}'#g" Jenkinsfile.release
```

Comment:
Jenkins script updates are out of scope for PAN-2485.  Please remove this change in this PR.